index.html

```diff
@@ -428,6 +438,22 @@ <h2><dfn>disconnect()</dfn> method</h2>
         "WHATWG-DOM#context-object">context object</a>'s <a>observer
         buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>supportedEntryTypes</dfn> attribute</h2>
```

Comment:
I don't think 2 `<dfn>` to the same attribute, so this should probably be an `<a>` (with the dfn being in the paragraph below)

Reply:
Changed the dfn to the place where attribute getter is described. However, there's not quite repetition. There's `supportedEntryTypes`, the attribute, and `supported entry types`, the list maintained by the user agent which is used as the return value.